django_netjsongraph/base/link.py

```diff
@@ -37,9 +38,21 @@ class AbstractLink(TimeStampedEditableModel):
                            load_kwargs={'object_pairs_hook': OrderedDict},
                            dump_kwargs={'indent': 4})
 
+    link_status_change = Signal(providing_args=["topology", "id"])
```

Comment:
I think signals are not usually defined as class attributes, so I think you can move this before the model definition.

Also, I would prefer `link_status_changed`.